hal/armv7m/imxrt/10xx/console.c

```diff
@@ -71,11 +80,21 @@ void console_init(void)
 	*(halconsole_common.uart + uart_global) &= ~(1 << 1);
 	imxrt_dataBarrier();
 
-	/* Set 115200 baudrate */
-	t = *(halconsole_common.uart + uart_baud);
-	t = (t & ~(0x1f << 24)) | (0x4 << 24);
-	*(halconsole_common.uart + uart_baud) = (t & ~0x1fff) | 0x8b;
-	*(halconsole_common.uart + uart_baud) &= ~(1 << 29);
+	/* Set baud rate */
+	t = *(halconsole_common.uart + uart_baud) & ~((0x1f << 24) | (1 << 17) | 0x1fff);
+
+	/* For baud rate calculation, default UART_CLK=80MHz was assumed */
+	switch (CONSOLE_BAUD(UART_CONSOLE)) {
+		case 9600: t |= 0x0c000281; break;
+		case 19200: t |= 0x080001cf; break;
+		case 38400: t |= 0x03020209; break;
+		case 57600: t |= 0x0302015b; break;
+		case 115200: t |= 0x0402008b; break;
+		case 230400: t |= 0x1c00000c; break;
+		case 460800: t |= 0x1c000006; break;
+		default: t |= 0x0402008b; break;
```

Comment:
What is set in default case? Comment would be helpful

Reply:
Added comment about default fallback value.